### DISCUSSION TITLE
IOS Testflight Crash logs

### DISCUSSION THREAD

**Discussion by Bnjbel97** [dotnet/maui#22535] [General]
I hope this is the right place to post this.  I have scanned for information on this issue, however I have not had much luck in answers. 

To describe the issue first.  The app is currently latest Maui build 8.0.40.  I have a beta app within testflight.  I have crash logs from testflight submitted by testers on a particular issue I am trying to trace.  I can access the crash logs however I do not see info that helps me trace back the issue well enough.  I believe the log needs to be symbolicated, however I'm not certain how to do this or what this will give me.  Below are a few things I have gone through but still no luck nor sure where to proceed. 

- Opened crash log in xcode. Gives some info like thread and generic crash. In this particular case it is thread 0 - bad address. This happens on background to foreground from user feedback.
- attempted to click symoblicate within xcode - says it is missing the dsym
- I have looked at the archive to download the symbols however it says they don't exist.
- checked on testflight under meta data says it does not have symbols. 
- below is from the project file release build - 
 		<RuntimeIdentifier>ios-arm64</RuntimeIdentifier>
		<ProvisioningType>manual</ProvisioningType>

		<MtouchProfiling>True</MtouchProfiling>
		<MtouchLink>SdkOnly</MtouchLink>
		<MtouchDebug>True</MtouchDebug>
		<Optimize>True</Optimize>
		<MtouchUseLlvm>True</MtouchUseLlvm>
		<EnableAssemblyILStripping>false</EnableAssemblyILStripping>
		<MtouchNoSymbolStrip>True</MtouchNoSymbolStrip>
		<EnableSGenConc>True</EnableSGenConc>
		<NoSymbolStrip>true</NoSymbolStrip>

I've tried to disable what options for symbol stripping I could find.  None appear to be what I'm looking for it seems. 

My questions 
1) How do I get more detailed info from the crash logs in testflight or the device itself.
2) What entry is needed to allow for debug symbols 
3) Is there another way to get the dsym from the Visual Studios side.  If so how do I supply that to xcode to allow it to be symbolicated. 

Again just looking for information in general on how to go about resolving this issue using the crash logs is very much appreciated.

**Comment by WilliamWatterson86** [comment]
@Bnjbel97 did you ever figure out how to decode them?

**Reply by Bnjbel97** [reply]
Unfortunately no. Though I have not looked at this in quite some time.
